tutorials/Quantile_Loss.ipynb

```diff
@@ -0,0 +1,172 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantile Loss (Score)\n",
+    "\n",
+    "[Geinting, T. (2011). Making and evaluating point forecasts, Journal of the American Statistical Association, 106(494), 746-762.](https://www.tandfonline.com/doi/abs/10.1198/jasa.2011.r10138)\n",
+    "\n",
+    "Quantile loss (score) is a consistent scoring function for an $\\alpha$-quantile forecast, where $\\alpha \\in (0, 1)$. Quantile loss function is also used in Machine Learning, in quantile regression, which focuses on estimating different quantiles of the conditional distributution of the response variable. Please note that quantiles are often referred to as value-at-risk in finance.\n",
+    "\n",
+    "The quantile loss function is defined as:\n",
+    "\n",
+    "$$ S(x, y) = \\begin{cases} \n",
+    "\\alpha \\cdot (y - x) & \\text{if } y \\geq x \\\\\n",
+    "(\\alpha - 1) \\cdot (y - x) & \\text{if } y < x \n",
+    "\\end{cases} $$\n",
+    "\n",
+    "where $S$ is the scoring function (here quantile loss), $x$ and $y$ are forecast and observation, respectively.\n",
+    "\n",
+    "For more information about quantile loss function, please read [Geinting, T. (2011)](https://www.tandfonline.com/doi/abs/10.1198/jasa.2011.r10138).\n",
+    "\n",
+    "\n",
+    "Now let's look at an example where we demonstrate how to use quantile score:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scores.probability import quantile_score\n",
+    "import numpy as np\n",
+    "import xarray as xr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we create synthetic forecast and observation data with dimensions of `('time', 'lat', 'lon')`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "forecast_data = np.random.rand(5, 3, 4)\n",
+    "observation_data = np.random.rand(5, 3, 4)\n",
+    "forecast = xr.DataArray(forecast_data, dims=('time', 'lat', 'lon'))\n",
+    "observation = xr.DataArray(observation_data, dims=('time', 'lat', 'lon'))"
```

Comment:
This is okay, but if you have time consider updating the tutorial to include two synthetic forecasts with one which is clearly better for targeting the 90th percentile, and the other one better at targeting the 50th percentile to make this tutorial a bit more interesting.

Reply:
As per your suggestion, two forecast systems have been included in the tutorial.